### DISCUSSION TITLE
Add ability to connect taxon tree to catalog of life or other sources

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5657] [Feature Requests]
It would be nice to use an external authority as the source for a local taxon tree, especially for collections who wish to refer only to an external authority.

There would need to be some sort of UI to reconcile the local taxon tree with the other source, and taxon records would need to be identified by GUIDs across every source. This is a very complicated thing to implement but it has been requested by many. How to handle when a tree updates, how to preserve modifications, and the many other follow up questions would need to be discussed.

**Comment by maxpatiiuk** [comment]
Per dev group meetings, this should be possible in Specify 7 without breaking compatibility with Specify 6.
See more in https://docs.google.com/document/d/1SmoTdB2LNOJ2gFilrEg_q1pkjSyGp35qSZ2_PFptQu8/edit#heading=h.ymg2vgs96xf8

**Comment by acbentley** [comment]
Creating the linkage is "easy".  The hard part is the messaging system that would be required to alert collections of impending changes to CO's in their collection based on the changes and the underlying business rules associated with creating new determinations from taxa changes.  You would also need to allow collections to accept or reject changes (I would think) in which case you would have the possible scenario of future changes being reliant on a previous change that had not been accepted.  Makes for a possible messy situation.

**Comment by grantfitzsimmons** [comment]
Northeastern is requesting bulk updating taxon trees based on an authority

**Comment by grantfitzsimmons** [comment]
New Brunswick is requesting this ability as well
